src/main/java/com/semmle/jira/addon/LgtmServlet.java

```diff
@@ -111,7 +111,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
         applyTransition(issue, Constants.WORKFLOW_CLOSE_TRANSITION_NAME, resp, config);
         break;
       case SUPPRESS:
-        if (!issue.getStatusId().equals(falsePositiveStatus.getId())) {
+        if (!issue.getStatusId().equals(falsePositiveStatus.getId())) { // LGTM
```

Comment:
I think you need to specify the rule for this to do anything... 

```// lgtm [java/dereferenced-value-may-be-null]```

Reply:
It's not needed but I'll add it

Reply:
Huh. Then I have been misinformed!